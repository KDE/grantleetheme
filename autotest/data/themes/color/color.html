<html>
Color name: {{ pal.button|colorHexRgb }}
RGBA: {{ pal.button|colorCssRgba }}
Color lighter: {{ pal.button|colorLighter:150|colorHexRgb }}
Color darker: {{ pal.button|colorDarker:150|colorHexRgb }}
Color set alpha: {{ pal.button|colorSetAlpha:42|colorCssRgba }}

Active: {{ pal.activeBase|colorHexRgb }}
Inactive: {{ pal.inactiveBase|colorHexRgb }}
Disabled: {{ pal.disabledBase|colorHexRgb }}
Default: {{ pal.base|colorHexRgb }}

Red: {{ pal.button.red }}
Name: {{ pal.button.hexRgb }}
RGBA: {{ pal.button.cssRgba }}

Mix literal/literal: {% colorMix "#ff0000" "#0000ff" 0.75 %}
Mix var/literal: {% colorMix pal.button "#0000ff" 0.5 %}
Mix var/var: {% colorMix pal.button pal.inactiveBase 0.25 %}

Color literal: {{ "lightsteelblue"|colorHexRgb }}

Mix filtered: {% colorMix pal.button "#0000ff" 0.5 as myColor %}{{ myColor|colorDarker:200|colorCssRgba }}

## Color scheme
Normal background: {{ colorScheme.normalBackground.hexRgb }}
Link background: {{ colorScheme.linkBackground.hexRgb }}
Normal text: {{ colorScheme.normalText.hexRgb }}
Link text: {{ colorScheme.linkText.hexRgb }}
Hover color: {{ colorScheme.hoverColor.hexRgb }}
Focus color: {{ colorScheme.focusColor.hexRgb }}

</html>
