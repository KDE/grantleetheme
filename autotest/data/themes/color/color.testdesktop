[Desktop Entry]
Name=Color Unit Test
Description=Color Unit Test
FileName=color.html
