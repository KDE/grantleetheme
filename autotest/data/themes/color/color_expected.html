<?xml version="1.0" encoding="UTF8"?>
<html>
Color name: #ff0000
RGBA: rgba(255, 0, 0, 1)
Color lighter: #ff7f7f
Color darker: #aa0000
Color set alpha: rgba(255, 0, 0, 0.164706)

Active: #ff0000
Inactive: #00ff00
Disabled: #0000ff
Default: #00ff00

Red: 255
Name: #ff0000
RGBA: rgba(255, 0, 0, 1)

Mix literal/literal: rgba(64, 0, 191, 1)
Mix var/literal: rgba(128, 0, 128, 1)
Mix var/var: rgba(191, 64, 0, 1)

Color literal: #b0c4de

Mix filtered: rgba(64, 0, 64, 1)

## Color scheme
Normal background: #fcfcfc
Link background: #e4eef7
Normal text: #232627
Link text: #2980b9
Hover color: #93cee9
Focus color: #3daee9

</html>
